en/app.ftl

```diff
@@ -1143,3 +1143,17 @@ forwarded-email-footer-offer-countdown-banner = Upgrade to { $premium_link } bef
 ## Notifications component
 
 toast-button-close-label = Close notification
+
+## API error messages
+
+# Variables:
+#   $free_tier_limit (string) - Maxmimum email masks created for free account
```

Comment:
Is this a string or a number?

Reply:
I had this PR as a draft. I should've added some sort of "Not ready for review" label. 🙈